### PR TITLE
merging branches

### PR DESCRIPTION
**Pull request by Lovj-web**
Updates to the following:
Navbar page
Profile page
stretch category page

### CODE REVIEW

**Review by saritasalazar — CHANGES_REQUESTED** [review]
Hey Love! It doesn’t look like any files are being added/changed in this PR. Make sure you have saved your changes and then run a ‘git status’ , ‘git add ‘ the changes you would like to stage and then ‘git commit -m “your msg here” ‘ and push back up to your branch. Let me know if you have any questions!

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+import { Link } from 'react-router-dom';
+import { Navbar, Nav, Dropdown, Image } from 'react-bootstrap';
+import { AppContext } from '../context/Appcontext';
+import Logout from './Logout';
 
 const Navbar = () => {
-  return <div></div>;
+  const { currentUser } = useContext(AppContext);
+
+  return;
```
Comment (saritasalazar) [inline client/src/components/Navbar.js:10]: Hey Love! You will want to get rid of the semi-colon and everything you are returning after your return should be wrapped in parenthesis ( )

File: client/src/components/NavbarSticky.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Navbar, Nav, Dropdown, Image } from 'react-bootstrap';
+import { AppContext } from '../context/Appcontext';
+import Logout from './Logout';
+
+const Navbar = () => {
+  const { currentUser } = useContext(AppContext);
+
+  return;
```
Comment (saritasalazar) [inline client/src/components/NavbarSticky.js:10]: Hey love! Same with this one, you will want remove the semicolon in line 10 and make sure you are wrapping in the contents of your return in ( )

**Review by saritasalazar — CHANGES_REQUESTED** [review]
Hey Love! There are just a few small changes that need to be made. Let me know if you have any questions!!

**Review by naaltunian — DISMISSED** [review]

**Review by naaltunian — APPROVED** [review]

**Review by ericshuanders — APPROVED** [review]

**Review by ljgalaty — APPROVED** [review]

**Pull request merged**